start

```diff
@@ -30,10 +30,15 @@ while :; do
 done
 
 cd "${projectRoot}"
-./ffc-pay-batch-processor/scripts/start -d && \
+  ./ffc-pay-batch-processor/scripts/start -d && \
   ./ffc-pay-enrichment/scripts/start -d && \
   ./ffc-pay-processing/scripts/start -d && \
   ./ffc-pay-submission/scripts/start -d && \
   ./ffc-pay-responses/scripts/start -d && \
   ./ffc-pay-web/scripts/start -d && \
-  ./ffc-pay-request-editor/scripts/start -d
+  ./ffc-pay-request-editor/scripts/start -d && \
+  ./ffc-pay-batch-validator/scripts/start -d && \
+  ./ffc-pay-file-sender/scripts/start -d && \
+  ./ffc-pay-file-consumer/scripts/start -d && \
+  ./ffc-pay-event/scripts/start -d && \
+  ./ffc-pay-alerts/scripts/start -d
```

Comment:
Don't think we can do this as starting functions will block the terminal.  Need to find a way to run functions detached.

Reply:
Yup, scan of `func` CLI and internet confirms this. I've created a workaround, check it out: https://github.com/DEFRA/ffc-pay-alerts/pull/4

Reply:
Work around seems to work ok so seems the way to go.  We can confirm once we've got them all updated.

Couple of conflicts with ports between `ffc-pay-alerts` and `ffc-pay-batch-validator` as both want to bind `7070` and `10004` which we'll need to resolve to use these scripts.  (and run them together in general)